src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -552,6 +552,7 @@ private class Emit(
         val eti = typeToTypeInfo(tarray.elementType)
         val xmv = mb.newField[Boolean](name2 + "_missing")
         val xvv = coerce[Any](mb.newField(name2)(eti))
+        val xmbody = mb.newField[Boolean](name1 + "_missing")
```

Comment:
I think this is acting as a temp variable. Would it make sense for it to have a distinct name from that of xmaccum?

Reply:
yeah, I can rename.